src/components/PopupWithForm.js

```diff
@@ -16,8 +17,17 @@ export default class PopupWithForm extends Popup {
     return data;
   }
 
+  setSubmitText(submit, submitText = "Saving...") {
```

Comment:
Great that you've implemented the code for rendering loading via the class methods